File: Projeto_Benner_WebAPI/Models/TabelaPreco.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Projeto_Benner_WebAPI.Models 
{
    public class TabelaPreco 
    {
        public TabelaPreco() {}
        public TabelaPreco (int id, DateTime vigenciaInicial, DateTime vigenciaFinal, float valorHora) 
        {
            this.Id = id;
            this.VigenciaInicial = vigenciaInicial;
            this.VigenciaFinal = vigenciaFinal;
            this.ValorHora = valorHora;
        }

        [Key]
        public int Id { get; set; }
        public DateTime VigenciaInicial { get; set; }
        public DateTime VigenciaFinal { get; set; }
        public float ValorHora { get; set; }
        public IEnumerable<Estacionamento> Estacionamento { get; set; }

    }
}

File: ProjetoBennerAngular/src/app/models/TabelaPreco.ts
export class TabelaPreco {
  constructor() {
    this.id = 0;
    this.vigenciaInicial = null;
    this.vigenciaFinal = null;
    this.valorHora = '';
  }

  id: number;
  vigenciaInicial: Date;
  vigenciaFinal: Date;
  valorHora: string;
}
